run.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import configparser
+import os
+import sys
+
+from geobeam.simulations import SimulationSetBuilder
+from geobeam.generate_route import TimedRoute
+from geobeam import gps_utils
+
+DEFAULT_SPEED = 1.4  # meters/sec
+DEFAULT_FREQUENCY = 10  # Hz
```

Comment:
These are only used as strings, so should they be string literals? Maybe with a comment to say that they're used in the config to explain why.